egs2/librispeech/asr2/run.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+
+kmeans_feature="wavlm_large/21"  # use model_type/layer_index
+nclusters=2000
+
+src_lang=$(echo "${kmeans_feature}_km${nclusters}" | tr "/" "_")
+tgt_lang=en
+
+train_set="train_960"
+train_dev="dev"
+test_sets="test_clean test_other dev_clean dev_other"
+
+asr_config=conf/tuning/train_discrete_asr_e_branchformer1.yaml
+inference_config=conf/decode_ctc0.3.yaml
+
+src_nbpe=6000   # I use src_nbpe=6000 for 2000-cluster kmeans.
+tgt_nbpe=5000   # if token_joint is True, then only tgt_nbpe is used
+
+# ts: true sequence
+# rm: deduplicated sequence which removes duplicated tokens
+src_case="rm"
+tgt_case="ts"
+
+./asr2.sh \
+    --kmeans_feature "${kmeans_feature}" \
+    --nclusters "${nclusters}" \
+    --ngpu 2 \
```

Comment:
As @ftshijt suggested, I removed the `nj` and `inference_nj` which used the default values.